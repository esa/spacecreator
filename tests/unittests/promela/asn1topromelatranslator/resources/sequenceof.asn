SequencesOf DEFINITIONS ::= BEGIN

Type1 ::= INTEGER(0..2)

SimpleFixedSizeSequenceOf ::= SEQUENCE (SIZE(4)) OF Type1

END