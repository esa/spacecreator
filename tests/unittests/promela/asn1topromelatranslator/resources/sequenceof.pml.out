inline Type1_generate_value(value)
{
    if
    ::  true;
        value = 0;
    ::  true;
        value = 1;
    ::  true;
        value = 2;
    fi;
}
inline SimpleFixedSizeSequenceOf_generate_value(value)
{
    int i;
    for(i : 0 .. 4)
    {
        Type1_generate_value(value.data[i]);
    }
}
